vo_new/components/reports/report-instance-view.tsx

```
@@ -0,0 +1,327 @@
+              <CardHeader>
+                <CardTitle className="flex items-center gap-2 text-blue-600">
+                  <Loader2 className="h-5 w-5 animate-spin" />
+                  Generating Report
+                </CardTitle>
+              </CardHeader>
+              <CardContent>
+                <div className="bg-blue-50 border border-blue-200 rounded-lg p-4">
+                  <p className="text-blue-800 mb-2">Your report is currently being generated.</p>
+                  <p className="text-blue-600 text-sm">
+                    This process typically takes 2-5 minutes depending on the dashboard complexity and data volume.
+                    You'll receive an email notification when the report is ready.
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)